DESCRIPTION

```diff
@@ -14,8 +14,12 @@ Imports:
   jaspBase,
   jaspGraphs,
   jaspDistributions,
-  ggforce
+  jaspDescriptives,
```

Comment:
why include jaspDescriptives? I don't see it being used in the code?

Reply:
It is in LSTDescriptives.R on line 654. Forgot to load it previously.